pipelines/src/main/scala/dagr/pipelines/ParallelHaplotypeCaller.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package dagr.pipelines
+
+import java.nio.file.{Files, Path}
+
+import dagr.core.cmdline.Pipelines
+import dagr.core.tasksystem.Task
+import dagr.pipelines.ParallelHaplotypeCaller.VariantCallTask
+import dagr.sopt._
+import dagr.tasks.DagrDef._
+import dagr.tasks.gatk.{GenotypeGvcfs, HaplotypeCaller, SplitIntervalsForCallingRegions}
+import dagr.tasks.misc.DeleteFiles
+import dagr.tasks.parallel.PathBasedScatterGatherPipeline
+import dagr.tasks.parallel.PathBasedScatterGatherPipeline.{GatherPathTask, ScatterPathTask}
+import dagr.tasks.picard.{FilterVcf, GatherVcfs}
+
+object ParallelHaplotypeCaller {
+
+  /** Runs GenotypeGvcfs, HaplotypeCaller, and optionally FilterVcf on a given interval list. */
+  private class VariantCallTask(ref: PathToFasta,
+                                bam: PathToBam,
+                                filterVcf: Boolean = true,
+                                dbsnp: Option[PathToVcf] = None,
+                                tmpDirectory: Option[Path] = None,
+                                intervals: PathToIntervals) extends ScatterPathTask {
+    var _output: Option[Path] = None
```

Comment:
Why not have this be a val and initialize it inline instead of in `getTasks`?  Or just have it specified by the caller?